sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Diagnostics/DiagnosticProperty.cs

```diff
@@ -46,24 +46,24 @@ internal static class DiagnosticProperty
         ///   The activity name associated with events.
         /// </summary>
         ///
-        public static string EventActivityName { get; } = $"{ BaseActivityName }.Message";
```

Comment:
nit:  Should we remove `BaseActivityName` entirely?   It doesn't look to be used outside of this class, and may help to avoid misuse in the future.